backend/Auth/ResetPassword.ts

```diff
@@ -115,6 +119,63 @@ router.get<Params, ResponseBody, QueryParams>(
   }
 );
 
+router.put(
+  "/api/passwords",
+  body("password").isStrongPassword(),
+  body("token").exists(),
```

Comment:
Slightly better to validate that the token is a string with a length greater than 0:

```suggestion
  body("token").notEmpty(),
```